src/client/pages/tocheckitem.react.js

```diff
@@ -0,0 +1,21 @@
+import Component from '../components/component.react';
+import React from 'react';
+import {FormattedHTMLMessage} from 'react-intl';
+
+class ToCheckItem extends Component {
+
+  render() {
+    return (
+      <li>
+        <FormattedHTMLMessage message={this.props.message} />
+      </li>
+    );
+  }
+
+}
+
+ToCheckItem.propTypes = {
+  message: React.PropTypes.string.isRequired
```

Comment:
What about messages with placeholders and pluralization? This one seems pretty simple and explicit. I'm not sure I fully get the concept of all things in new este. If you see some technologically pure solution, please suggest it.